packages/gatsby-theme-store/src/sdk/buyButton/useBuyButton.ts

```diff
@@ -73,6 +89,20 @@ export const useBuyButton = ({
       return
     }
 
+    const pixelEventItem = {
+      productId: product?.id,
+      productReferenceId: product?.productReference,
+      productName: product?.productName ?? productName,
+      brand: product?.brand,
+      categoryTree: product?.categoryTree,
+      price: seller.commercialOffer.price,
+      // TODO currencyCode,
```

Comment:
Should this comment go to production?

Reply:
Yes, it should. When we start supporting multiple currencies and etc we should send it with these events, but I thought it was better to not worry about that now.